searx/engines/odysee.py

```diff
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Odysee_ is a decentralised video hosting platform.
+
+.. _Odysee: https://github.com/OdyseeTeam/odysee-frontend
+"""
+
+import time
+from datetime import datetime
+
+# Engine metadata
+about = {
+    "website": "https://odysee.com/",
+    "wikidata_id": "Q102046570",
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+# Engine configuration
+paging = True
+results_per_page = 20
+categories = ['images']
```

Comment:
I removed the timeout from the settings.yml .. the default timeout has always been enough / I did not had any issues with timeouts. And I moved the definition of the categories from settings.yml file to here in the odysee.py module.